draft-demarco-oauth-status-assertions.md

```diff
@@ -378,14 +383,14 @@ and payload are presented without applying signature and encoding for better rea
        / iat    / 6: 1698744039 /,
        / exp    / 4: 1698830439 /,
        / cti    / 7: 6f204f7e-e453-4dfd-814e-9d155319408c /,
-       / credential_hash / 8: $Issuer-Signed-JWT-Hash /,
+       / credential_hash / 8: $hash-about-MobileSecurityObject /,
```

Comment:
in CBOR, this structure might be a better solution for this:

```
COSE_X509 = bstr / [ 2*certs: bstr ]
COSE_CertHash = [ hashAlg: (int / tstr), hashValue: bstr ]
```

See https://datatracker.ietf.org/doc/html/rfc9360

Reply:
issue here: https://github.com/peppelinux/draft-demarco-oauth-status-assertions/issues/72

I admit that I didn't get it, we'll continue the conversation in the issue